content/docs/gke/deploy/deploy-cli.md

```diff
@@ -70,7 +71,7 @@ Follow these steps to deploy Kubeflow:
     kfctl init ${KFAPP} --platform gcp --project ${PROJECT} --use_basic_auth -V
 
     cd ${KFAPP}
-    kfctl generate all -V
+    kfctl generate all -V --zone ${ZONE}
```

Comment:
@gabrielwen Can people specify the zone in the `kfctl init` command or only in the `generate` command?

Reply:
no - location is not needed during `init`

Reply:
Great, thanks Gabriel.